src/components/List/List.jsx

```diff
@@ -5,20 +5,45 @@ import { useFirebase } from '../../hooks/useFirebase';
 const List = () => {
   const token = localStorage.getItem('token');
 
-  const { getAll } = useFirebase();
+  const { getAll, update } = useFirebase();
 
   const firebasePath = getAll().doc(token).collection('items');
 
   const [value, loading, error] = useCollection(firebasePath);
 
+  const handleCheck = (id, lastDate) => {
+    if (isPassed24Hours(lastDate) === false) {
+      const date = new Date();
+      update(token, id, { lastDate: date });
+    }
+  };
+
+  const isPassed24Hours = (date) => {
+    if (date) {
+      const currentDate = new Date();
+      const itemDate = new Date(date.toDate());
+      itemDate.setDate(itemDate.getDate() + 1);
+      // itemDate.setMinutes(itemDate.getMinutes() + 2); // In case you wanna test it
+
+      return !(currentDate >= itemDate);
+    }
+    return false;
+  };
```

Comment:
can you add space here :D

Reply:
I added a space in line 29, not sure if that's what you meant 🤔